client/app/lib/components/ProviderWrapper.jsx

```diff
@@ -129,17 +129,7 @@ const ProviderWrapper = ({ store, persistor, children }) => {
   let providers = (
     <>
       {children}
-      <ToastContainer
-        position="bottom-center"
-        autoClose={5000}
-        hideProgressBar={false}
-        newestOnTop={false}
-        closeOnClick
-        rtl={false}
-        pauseOnFocusLoss
-        draggable
-        pauseOnHover
-      />
+      <ToastContainer position="bottom-center" />
```

Comment:
The popper issue was caused by toast config?

Reply:
No. I removed these props because they were basically [default values](https://fkhadra.github.io/react-toastify/api/toast-container); trying to keep the code lean. Apologies for the confusion ><